### PR TITLE
Connect backend with frontend

### PR DESCRIPTION
**Pull request by Matt-1985**
I try to display my data from backend in my card component

### CODE REVIEW

File: client/src/components/Card.js [hunk 1]
```diff
@@ -29,14 +29,25 @@ const DesicionInfo = styled.p`
...
 `;
 
-export default function Card() {
+export default function Card({
+  restaurantImg,
+  restaurantName,
+  restaurantDiet,
+  restaurantCuisine,
+}) {
+  const diet = restaurantDiet.toString();
```
Comment (phlgr) [inline client/src/components/Card.js:38]:
```suggestion
 restaurant
}) {
  const diet = restaurantDiet.toString();
```

File: client/src/components/Card.js [hunk 2]
```diff
@@ -29,14 +29,25 @@ const DesicionInfo = styled.p`
...
-      <DesicionImage src={restaurant} alt="restaurant" />
-      <DesicionHeader>LOREM IPSUM</DesicionHeader>
-      <DesicionInfo>
-        Est duis labore dolor ut consectetur eu do ex anim tempor.
-      </DesicionInfo>
+      <DesicionImage src={restaurantImg} alt="restaurant" />
+      <DesicionHeader>{restaurantName}</DesicionHeader>
+      <DesicionInfo>{`${diet}, ${restaurantCuisine}`}</DesicionInfo>
     </DesicionCardContainer>
   );
```
Comment (phlgr) [inline client/src/components/Card.js:45]: I would probably pass the whole object and destructure it in here so you have less points of failure.

File: client/src/components/Card.js [hunk 3]
```diff
@@ -29,14 +29,25 @@ const DesicionInfo = styled.p`
...
   return (
     <DesicionCardContainer>
-      <DesicionImage src={restaurant} alt="restaurant" />
-      <DesicionHeader>LOREM IPSUM</DesicionHeader>
-      <DesicionInfo>
-        Est duis labore dolor ut consectetur eu do ex anim tempor.
-      </DesicionInfo>
+      <DesicionImage src={restaurantImg} alt="restaurant" />
+      <DesicionHeader>{restaurantName}</DesicionHeader>
+      <DesicionInfo>{`${diet}, ${restaurantCuisine}`}</DesicionInfo>
```
Comment (phlgr) [inline client/src/components/Card.js:43]: e.g.
```suggestion
      <DesicionImage src={restaurant.img} alt="restaurant" />
      <DesicionHeader>{restaurant.name}</DesicionHeader>
      <DesicionInfo>{`${restaurant.diet}, ${restaurant.cuisine}`}</DesicionInfo>
```

File: client/src/components/Card.js [hunk 4]
```diff
@@ -29,14 +29,25 @@ const DesicionInfo = styled.p`
...
     </DesicionCardContainer>
   );
 }
+
+Card.propTypes = {
+  restaurantImg: PropTypes.string,
+  restaurantName: PropTypes.string,
+  restaurantDiet: PropTypes.array,
+  restaurantCuisine: PropTypes.string,
```
Comment (phlgr) [inline client/src/components/Card.js:52]: You could then reduce this to this:
```suggestion
  restaurant: PropTypes.object.isRequired
```

File: client/src/components/MultiSelectBox.js [hunk 5]
```diff
@@ -33,6 +53,21 @@ const MultiSelectBox = () => {
...
         className="multiselect"
       />
+      {filteredRestaurants?.map((filteredRestaurant) => (
+        <Card
+          key={filteredRestaurant._id}
+          restaurantImg={filteredRestaurant.img}
+          restaurantName={filteredRestaurant.name}
+          restaurantDiet={filteredRestaurant.diet}
+          restaurantCuisine={filteredRestaurant.cuisine}
+        />
```
Comment (phlgr) [inline client/src/components/MultiSelectBox.js:60]: Why don't you just pass the restaurant object into here? Would make your code much slimmer
```suggestion
        <Card
          restaurant={filteredRestaurant}
        />
```

**Review by phlgr — APPROVED** [review]
These are just suggestions and there are both down and upsides to both solutions. For easier maintaining it would be better to just pass the restaurant, but the readability is higher with you approach

File: client/src/components/MultiSelectBox.js (hunk 5 above)
Comment (Matt-1985) [reply client/src/components/MultiSelectBox.js:60]: Thank you very much for the review! i'll get to it during my refactoring 🦾 I have it on my toDo list

**Pull request merged**